src/main/kotlin/stringcalculator/Operator.kt

```diff
@@ -0,0 +1,28 @@
+package stringcalculator
+
+enum class Operator(
+    private val operatorSign: String,
+    private val operation: (Int, Int)
+    -> Int
```

Comment:
```suggestion
    private val operation: (Int, Int) -> Int,
```
람다를 정의할 때는 한 줄로 타입을 정의해주세요!
또, 코틀린에서는 trailing comma를 문법적으로 지원합니다 :)